client/web/api.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package web
+
+import (
+	"io"
+	"net/http"
+	"strings"
+
+	"tailscale.com/client/tailscale"
+	"tailscale.com/client/tailscale/apitype"
+)
+
+type apiHandler struct {
```

Comment:
do we need the extra apiHandler type?  What if we just put the *tailscale.LocalClient directly on the main WebClient type (or whatever we call it)?

Reply:
yeah fine with that, probably don't need the extra layer of abstraction